UserDetailsClient/UserDetailsClient.Core/MainPage.xaml.cs

```diff
@@ -8,19 +8,9 @@ namespace UserDetailsClient.Core
 {
     public partial class MainPage : ContentPage
     {
-        protected readonly IAuthenticationService authenticationService;
```

Comment:
There is no point in having this Interface, because the `B2CAuthenticationService` is not platform specific. I wrapper the platform specific bits in `IParentWindowLocatorService`